build.sc

```diff
@@ -555,11 +555,8 @@ trait VectorSubsystem
         "--split-verilog",
         "--strip-debug-info",
         "--preserve-values=none",
-        "--preserve-aggregate=all",
```

Comment:
why removing this, if there is some bug in the CIRCT, please open an issue there.